cinema/serializers.py

```diff
@@ -114,9 +114,9 @@ class TicketSerializer(serializers.ModelSerializer):
     def validate(self, attrs):
```

Comment:
In the `validate` method of `TicketSerializer`, you are calling `Ticket.validate_ticket`. This seems to be a custom validation method in the `Ticket` model. It's a good practice to put complex validation logic in the model, but it's also recommended to include basic validation in the serializer using field-level validation methods or using the `validators` option in the field definition. This way, the serializer can catch validation errors before it hits the model layer.